examples/server-side-rendering/src/app/app.component.ts

```diff
@@ -69,16 +88,18 @@ export class AppComponent {
     private injector: Injector,
     @Inject(PLATFORM_ID) private platformId: Object
   ) {
+    const routing = { router: history(), stateMapping: simple() };
+
     const req = isPlatformServer(this.platformId)
       ? this.injector.get('request')
       : undefined;
```

Comment:
This makes the logic easier.

```suggestion
       : { url: '' };
```